### PR TITLE
[bitnami/*] Add affinity to those charts that didn't have it previously

### PR DESCRIPTION
**Pull request by carrodher**
**Description of the change**

Add `affinity` to those charts that didn't have it previously. Tested in the following way:

- Edit `values.yaml`:
```yaml
## Affinity for pod assignment
## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
##
affinity:
  podAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
    - labelSelector:
        matchLabels:
          app.kubernetes.io/component: petete
      topologyKey: "kubernetes.io/hostname"
```
- Run `helm install . --name foobar`
- Check the pod:
```bash
$ kubectl get pods
NAME                              READY   STATUS             RESTARTS   AGE
foobar-apache-5979cbc5c6-xptkx    0/1     Pending            0          47m

$ kubectl describe pod foobar-apache-5979cbc5c6-xptkx
...
Events:
  Type     Reason             Age                     From                Message
  ----     ------             ----                    ----                -------
  Normal   NotTriggerScaleUp  27m (x17 over 47m)      cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added): 1 node(s) didn't match pod affinity rules, 1 node(s) didn't match pod affinity/anti-affinity
  Warning  FailedScheduling   22m (x141 over 48m)     default-scheduler   0/5 nodes are available: 5 node(s) didn't match pod affinity rules, 5 node(s) didn't match pod affinity/anti-affinity.
  Warning  FailedScheduling   8m9s (x20 over 20m)     default-scheduler   0/5 nodes are available: 1 Insufficient cpu, 4 node(s) didn't match pod affinity rules, 4 node(s) didn't match pod affinity/anti-affinity.
  Normal   NotTriggerScaleUp  3m1s (x231 over 48m)    cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added): 1 node(s) didn't match pod affinity/anti-affinity, 1 node(s) didn't match pod affinity rules
  Warning  FailedScheduling   2m39s (x13 over 4m59s)  default-scheduler   0/6 nodes are available: 2 Insufficient cpu, 4 node(s) didn't match pod affinity rules, 4 node(s) didn't match pod affinity/anti-affinity.
```

**Benefits**

Users can set `affinity`

**Checklist** [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by javsalgar** [comment]
I think the version should be minor

File: bitnami/mysql/templates/slave-statefulset.yaml [hunk 1]
```diff
@@ -196,3 +196,7 @@ spec:
           emptyDir: {}
 {{- end }}
 {{- end }}
+      {{- with .Values.affinity }}
```
Comment (javsalgar) [inline bitnami/mysql/templates/slave-statefulset.yaml]: I think that the affinity values should be different for master and slave

File: bitnami/metrics-server/templates/deployment.yaml [hunk 2]
```diff
@@ -39,3 +39,8 @@ spec:
       {{- range $key, $value := .Values.extraArgs }}
         - --{{ $key }}={{ $value }}
       {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+
```
Comment (miguelaeh) [inline bitnami/metrics-server/templates/deployment.yaml]: The rest of files haven't the empty line an the end, please remove it.

**Comment by miguelaeh** [comment]
LGTM!

**Pull request merged**